### DISCUSSION TITLE
Use secrets file for livestream username/password?

### DISCUSSION THREAD

**Discussion by moofur** [tsightler/ring-mqtt#751] [Support Q&A] [answered]
I'm running ring-mqtt in a docker stack separate from my Home Assistant environment (Home Assistant OS). Now, first off, I know I can run ring-mqtt as an add-on since I'm running HASS OS. I don't want to. I just today discovered an issue with my MQTT Broker add-on that caused me to roll my MQTT + Ring-MQTT stack over to a separate docker host where I originally had them running.

Now, with that out of the way...

I have the `livestream_user` and `livestream_pass` parameters set directly in my config.json file. Is it possible to offload these to a secrets file instead? Additionally, it would be swell if I could offload the username and password for my MQTT broker URL, as well, instead of having to have `mqtt://username:password@host:port` right there in the config.json file.

Not the end of the world if this isn't an option, but I figured it wouldn't hurt to ask.

**Comment by tsightler** [comment, marked as answer]
No, it's not possible and no current plans to support this.
